website/project/views/node.py

```diff
@@ -1063,7 +1066,7 @@ def project_private_link_edit(auth, **kwargs):
     private_link_id = request.json.get('pk', '')
     private_link = PrivateLink.load(private_link_id)
     if private_link:
-        private_link.name = new_name
+        private_link.name = sanitize(new_name)
```

Comment:
`website.util.sanitize` is a module, not a function. 

You're on the right track to use this module though: in general, framework.forms.util.sanitize() is deprecated in favor of `website.util.sanitize.strip_html()`. 

The first one allows some HTML through. The latter removes all HTML from the string. We do not want any HTML to be allowed in the name field of a private link.